conans/client/command.py

```diff
@@ -460,7 +460,7 @@ def package(self, *args):
             E.g. conan package MyPackage/1.2@user/channel 9cf83afd07b678da9c1645f605875400847ff3
         """
         parser = argparse.ArgumentParser(description=self.package.__doc__, prog="conan package")
-        parser.add_argument("reference", help='package recipe reference name. '
+        parser.add_argument("reference", nargs='?', default="", help='package recipe reference name. '
```

Comment:
Update the help from the argument and from the global package method. Explain the new use case